core/eolearn/tests/test_eodata_io.py

```diff
@@ -277,3 +282,56 @@ def test_lazy_loading_plus_overwrite_patch(fs_loader, folder_name, eopatch):
         lazy_eopatch.save(folder_name, filesystem=temp_fs, overwrite_permission=OverwritePermission.OVERWRITE_PATCH)
         assert temp_fs.exists(fs.path.join(folder_name, "data", "whatever.npy"))
         assert not temp_fs.exists(fs.path.join(folder_name, "data_timeless", "mask.npy"))
+
+
+def data_equality(data1, data2):
```

Comment:
perhaps rename into `assert_data_equal`. Also add type annotations (probably just Any and None).